src/lib.rs

```diff
@@ -1,4 +1,18 @@
-use sysinfo::{ComponentExt, System, SystemExt};
+use std::cmp;
+use sysinfo::{ComponentExt, PidExt, ProcessExt, System, SystemExt};
+
+/// Structure containing
+///
+/// `pid: u32`
+///
+/// `name: String`
+///
+/// `cpu_usage: f32`
+pub struct ProcessStruct {
```

Comment:
you should probably call this `ProcessInfo`, make the types name more descriptive; `struct ` as information is already included in the type